Simplified/MyBooks/NYPLMyBooksDownloadCenter+AudiobooksDownload.swift

```diff
@@ -9,8 +9,54 @@
 import Foundation
 import NYPLAudiobookToolkit
 
+extension NYPLMyBooksDownloadCenter {
+  /// Create an audiobook object and add to download queue
+  @objc(downloadAudiobookForBook:)
+  func downloadAudiobook(for book: NYPLBook?) {
+    guard let book = book,
+          let url = self.fileURL(forBookIndentifier: book.identifier) else {
+      return
```

Comment:
can you add a `Log.error` here? Make sure to log both `book.loggableShortString` and `url` so that we can know which one is nil